contracts/adapters/FlexibleLeverageStrategyAdapter.sol

```diff
@@ -526,13 +702,23 @@ contract FlexibleLeverageStrategyAdapter is BaseAdapter {
         return _shouldRebalance(currentLeverageRatio, _customMinLeverageRatio, _customMaxLeverageRatio);
     }
 
+    /**
+     * Gets the list of enabled exchanges
+     */
+    function getEnabledExchanges() external view returns (string[] memory) {
```

Comment:
We can probably just make the state variable public and remove this function

Reply:
This is here mostly to make testing that exchanges have been removed properly easier. Otherwise, you have to check if getting the nth element reverts (since there is no way to check the length of an array exposed via a public method)

Reply:
gotcha